docs/reference/sensors/hwpc-sensor.md

```diff
@@ -109,8 +150,7 @@ Table below depicts the different group events for compatible Intel and AMD arch
 }
 ```
 
-Once you have your configuration file, run HWPC Sensor using one of the following command lines, depending on the installation you use:
-
+The following CLI command shows how to use this configuration file in the deployment of an HWPC Sensor container :
```

Comment:
`HWPC Sensor ` instance vs `HWPC Sensor` container. Maybe to use `HWPC Sensor` instance everywhere and indicate that the instance here is deployed via docker (or a container)